src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -235,6 +235,12 @@ where automation.Name.Equals(calculateLibraryFuncName)
                
 
             stepUIPopup.IsOpen = true;
+
+            var nextButton = stepUIPopup.FindName("NextButton") as Button;
```

Comment:
anyway we can move this string somewhere?

Reply:
Yes, moved it to a readonly string.